base-template.sh

```diff
@@ -491,6 +491,19 @@ process_shared_hooks() {
     execute_shared_hooks "$@" || return 1
 }
 
+#####################################################
+# Check is not a supported git clone url and
+#   is treated as local path.
+# Returns:
+#   none
+#####################################################
+is_local_url() {
+    if echo "$1" | grep -Eq "(https?|file|ssh|ftp|git|file)://"; then
```

Comment:
Maybe simply check the protocol separator
```suggestion
    if echo "$1" | grep -Eq "[a-zA-Z]+://"; then
```

Reply:
Btw, this might be fun to test with our existing tests... Though I guess we can just switch them over to `file://`